### PR TITLE
Translation into Ukrainian language

### PR DESCRIPTION
**Pull request by KyloRen1**
#2782

### CODE REVIEW

**Comment by KyloRen1** [comment]
@karlhigley @iamtrask Hello! Is my pull request good to go?

**Comment by karlhigley** [comment]
Changed the base branch to `master`, rebased, and squashed commits.

File: examples/tutorials/translations/Ukrainian/Part 01 - Основні засоби приватного глибинного навчання.ipynb [hunk 1]
```diff
@@ -0,0 +1,960 @@
+{
```
Comment (karlhigley) [inline examples/tutorials/translations/Ukrainian/Part 01 - Основні засоби приватного глибинного навчання.ipynb]: <p>In other languages, we've been adding a second list for "translators" below the list of "authors." </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2854/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

**Comment by KyloRen1** [comment]
@karlhigley Seems like I have fixed the issue

**Comment by karlhigley** [comment]
Closing and re-opening to trigger CI again.

**Comment by karlhigley** [comment]
Travis builds aren't running for...reasons. I've contacted Travis support for their help resolving the issue.

**Comment by karlhigley** [comment]
@KyloRen1 Thanks so much PR, and apologies it's taken so long to get merged! We've really been struggling with Travis CI, so have migrated off of it and are finally able to get translations merged again.

**Pull request merged**